src/main/java/com/minecolonies/coremod/entity/pathfinding/Pathfinding.java

```diff
@@ -29,7 +29,7 @@
     private static final ThreadPoolExecutor executor;
     static
     {
-        executor = new ThreadPoolExecutor(1, Configurations.pathfinding.pathfindingMaxThreadCount, 10, TimeUnit.SECONDS, jobQueue);
+        executor = new ThreadPoolExecutor(1, Configurations.Pathfinding.pathfindingMaxThreadCount, 10, TimeUnit.SECONDS, jobQueue);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)